cmd/aries-js-worker/package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "@hyperledger/aries-framework-go",
   "version": "0.0.1",
```

Comment:
this version should be snapshot

Reply:
Aside from that - what should be the base version?

Reply:
(should we follow the general version of aries-framework-go or should this component be versioned separately?)

Reply:
It would make sense to follow general version of framework. @llorllale

Reply:
check now